design/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!--Google font-->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <!--Style sheet-->
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Mobile portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h2 class="logo">My Logo</h2>
+        <img src="./assets/Union.svg" alt="burger-menu" />
+      </nav>
+    </header>
+    <main>
+      <section id="hero-section">
+        <!-- <div class="section-main-img"></div> -->
```

Comment:
Great job @z4cope 🤝

- [ ] [Optional] Kindly remove commented code from your project as this is regarded as a bad practice.